stable/prometheus-blackbox-exporter/templates/prometheusrule.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
     app.kubernetes.io/managed-by: {{ $.Release.Service }}
     helm.sh/chart: {{ include "prometheus-blackbox-exporter.chart" $ }}
     {{- with .Values.prometheusRule.additionalLabels }}
-    {{- toYaml . | indent 4 }}
+    {{- toYaml . | nindent 4 }}
     {{- end }}
```

Comment:
I was still getting errors with this in my test.  I was able to solve it by using:
```
    {{- range $key, $val := .Values.prometheusRule.additionalLabels }}
    {{ $key }}: {{ $val | quote }}
    {{- end }}
```

Reply:
@jakemccloskey 
The suggestion by @desaintmartin works fine on my terminal.
My `value.yaml` for checking is:

```
prometheusRule:
  enabled: true
  additionalLabels: 
    release: foo
    bar: baz
```

Reply:
I was able to test again today and get my values.yaml working with `{{- toYaml . | nindent 4 }}` so it LGTM now!  Thanks @monaka!

Here's my test :
```
root@e799e0823d5e:/charts/stable/prometheus-blackbox-exporter# cat templates/prometheusrule.yaml
{{- if .Values.prometheusRule.enabled }}
apiVersion: monitoring.coreos.com/v1
kind: PrometheusRule
metadata:
  name: {{ template "prometheus-blackbox-exporter.fullname" . }}
  {{- with .Values.prometheusRule.namespace }}
  namespace: {{ . }}
  {{- end }}
  labels:
    app.kubernetes.io/name: {{ include "prometheus-blackbox-exporter.name" $ }}
    app.kubernetes.io/instance: {{ $.Release.Name }}
    app.kubernetes.io/managed-by: {{ $.Release.Service }}
    helm.sh/chart: {{ include "prometheus-blackbox-exporter.chart" $ }}
    {{- with .Values.prometheusRule.additionalLabels }}
    {{- toYaml . | nindent 4 }}
    {{- end }}
spec:
  {{- with .Values.prometheusRule.rules }}
  groups:
    - name: {{ template "prometheus-blackbox-exporter.name" $ }}
      rules: {{ tpl (toYaml .) $ | nindent 8 }}
  {{- end }}
{{- end }}
root@e799e0823d5e:/charts/stable/prometheus-blackbox-exporter# cat values.yaml
prometheusRule:
  enabled: true
  additionalLabels:
    release: 'prometheus'
    foo: 'bar'
  rules:
  - alert: Blackbox_Exporter_Probe_Failure
    expr: probe_success != 1
    for: 1m
    labels:
      severity: critical
    annotations:
      summary: Probe Failed.
root@e799e0823d5e:/charts/stable/prometheus-blackbox-exporter# helm template -f values.ya
ml .
---
# Source: prometheus-blackbox-exporter/templates/prometheusrule.yaml
apiVersion: monitoring.coreos.com/v1
kind: PrometheusRule
metadata:
  name: RELEASE-NAME-prometheus-blackbox-exporter
  labels:
    app.kubernetes.io/name: prometheus-blackbox-exporter
    app.kubernetes.io/instance: RELEASE-NAME
    app.kubernetes.io/managed-by: Helm
    helm.sh/chart: prometheus-blackbox-exporter-3.5.0
    foo: bar
    release: prometheus
spec:
  groups:
    - name: prometheus-blackbox-exporter
      rules:
        - alert: Blackbox_Exporter_Probe_Failure
          annotations:
            summary: Probe Failed.
          expr: probe_success != 1
          for: 1m
          labels:
            severity: critical
```